dotCMS/src/main/java/com/dotmarketing/common/db/DotConnect.java

```diff
@@ -542,6 +542,10 @@ public DotConnect addParam(java.util.Date x) {
      */
 
     public DotConnect addJSONParam(Object json) {
+        if(json==null) {
+            return addObject(json);
+        }
+
```

Comment:
PreConditions?

Reply:
This is a different case. I do want to do something which is adding null using `addObject`